bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/SingleDirectoryDbLedgerStorage.java

```diff
@@ -695,6 +696,18 @@ private void fillReadAheadCache(long orginalLedgerId, long firstEntryId, long fi
         }
     }
 
+    private boolean chargeReadAheadCache(int currentReadAheadCount, long currentReadAheadBytes) {
```

Comment:
can we add unit tests for this ? you can make it a static method, it will be very easy to write an unit test.
the logic is not straightforward, a test with edge cases will help

Reply:
> can we add unit tests for this ? you can make it a static method, it will be very easy to write an unit test.
> the logic is not straightforward, a test with edge cases will help

Thanks for reviewing , I will add unit test today @eolivelli

Reply:
> I saw the unit tests, very good.
> 
> But here I meant to add a "unit test" that tests this method (you can make it "static", no need to start a Bookie)

updated,thanks